scripts/upload.sh

```diff
@@ -44,5 +44,12 @@ yq '.Resources.* | select(has("Type") and .Type == "AWS::Serverless::LayerVersio
 echo "Zipping the CloudFormation template"
 zip template.zip cf-template.yaml
 
+METADATA_ARGS="repository=$GITHUB_REPOSITORY,commitsha=$GITHUB_SHA,committag=$GIT_TAG,commitmessage=$COMMIT_MSG,mergetime=$MERGE_TIME,skipcanary=$SKIP_CANARY_DEPLOYMENT,commitauthor='$GITHUB_ACTOR'"
+# Check if VERSION_NUMBER is set and append it to METADATA_ARGS
+if [ -n "$VERSION_NUMBER" ]; then
+    METADATA_ARGS="$METADATA_ARGS,release=$VERSION_NUMBER,codepipeline-artifact-revision-summary=$VERSION_NUMBER"
+fi
+
+# Upload the zipped CloudFormation artifact to S3
 echo "Uploading zipped CloudFormation artifact to S3"
```

Comment:
```suggestion
echo "Uploading zipped CloudFormation artifact to S3"
```
This is unnecessary since the following `echo` prints the exact same message